Makefile

```diff
@@ -259,6 +260,14 @@ lint:
 		bash -c "[[ ! -z '$${SEARCH_PATH}' ]] && find $${SEARCH_PATH} -name '*.py' | sort -u | xargs pylint --py3k" && \
 		echo "--- Linting done. ---"; \
 	fi
+	. $(VENVNAME)/bin/activate; \
+	git diff $(MASTER_BRANCH) --name-only | xargs isort -c --diff && \
+	echo "--- isort check done. ---;"
```

Comment:
I suggest to update this part to inform people about `make lint_fix`. E.g. like that:
```diff
diff --git a/Makefile b/Makefile
index c4d82dc7..64e56008 100644
--- a/Makefile
+++ b/Makefile
@@ -260,9 +260,16 @@ lint:
 		bash -c "[[ ! -z '$${SEARCH_PATH}' ]] && find $${SEARCH_PATH} -name '*.py' | sort -u | xargs pylint --py3k" && \
 		echo "--- Linting done. ---"; \
 	fi
+
 	. $(VENVNAME)/bin/activate; \
-	git diff $(MASTER_BRANCH) --name-only | xargs isort -c --diff && \
-	echo "--- isort check done. ---;"
+	git diff $(MASTER_BRANCH) --name-only | xargs isort -c --diff || \
+	{ \
+		echo; \
+		echo "------------------------------------------------------------------------------" \
+		echo "Hint: Apply the required changes."; \
+		echo "      Execute the following command to apply them automatically: make lint_fix"; \
+		exit 1; \
+	} && echo "--- isort check done. ---";
 
 lint_fix:
 	. $(VENVNAME)/bin/activate; \
```


The output looks then something like this:
```
ERROR: /home/pstodulk/special/leapp/leapp-repository/repos/system_upgrade/el7toel8/actors/checkgrep/actor.py Imports are incorrectly sorted.
--- /home/pstodulk/special/leapp/leapp-repository/repos/system_upgrade/el7toel8/actors/checkgrep/actor.py:before	2021-12-02 10:39:32.098772
+++ /home/pstodulk/special/leapp/leapp-repository/repos/system_upgrade/el7toel8/actors/checkgrep/actor.py:after	2021-12-02 10:59:27.462197
@@ -1,9 +1,9 @@
+from leapp import reporting
 from leapp.actors import Actor
 from leapp.libraries.common.rpms import has_package
 from leapp.models import InstalledRedHatSignedRPM
-from leapp.reporting import Report, create_report
+from leapp.reporting import create_report, Report
 from leapp.tags import ChecksPhaseTag, IPUWorkflowTag
-from leapp import reporting
 
 
 class CheckGrep(Actor):

------------------------------------------------------------------------------ echo Hint: Apply the required changes.
      Execute the following command to apply them automatically: make lint_fix
make: *** [Makefile:249: lint] Error 1
```

Reply:
Applied, thanks!